src/Streamly/Internal/Unicode/Stream.hs

```diff
@@ -15,6 +15,8 @@ module Streamly.Internal.Unicode.Stream
       decodeLatin1
 
     -- ** UTF-8 Decoding
+    , writeCharUtf8'
+    , parseCharUtf8D'
```

Comment:
Need to export parseCharUtf8 i.e. Parser type and not a ParserD type routine.

Reply:
Will do. I'm exporting `parseCharUtf8D'` as well for benchmarking.